LOCKED_DEPS.md

```diff
@@ -0,0 +1,24 @@
+# Locked deps
+
+This is a file listing depencencies whose versions we had to lock as they were breaking.
```

Comment:
What does "locking a dependency" mean? If it means "only allowing a single version", then we're not locking prettier when we specify `~3.4.2`.